backend/src/errors.rs

```diff
@@ -0,0 +1,13 @@
+error_chain! {
+    links {
+    }
+
+    foreign_links {
+        ::std::io::Error, Io, "io error";
+        ::json::Error, Json, "json error";
+        ::chrono::ParseError, Chrono, "chrono parsing error";
```

Comment:
why do you use error-chain? I'd have thought it is unneccessary and we can just use simple errors. But if there is good motivation, then I'm ok with using it.